src/modules/RootPart.rb

```diff
@@ -1723,7 +1735,9 @@ def inject_intsys_files
       # resolver like systemd-resolver and some configuration of network manager. So we not modify
       # symlink target and instead just replace symlink with our file that can resolve and from
       # backup we later restore original symlink.
-      ::FileUtils.copy_entry(RESOLV_CONF, target, false, false, true) if File.exist?(RESOLV_CONF)
+      if File.symlink?(old_target)
```

Comment:
Note: `old_target` variable does not exist here. Probably a reference to `target`.